parley/src/context.rs

```diff
@@ -60,11 +60,31 @@ impl<B: Brush> LayoutContext<B> {
             .resolve_entire_style_set(font_ctx, raw_style, scale)
     }
 
+    /// Create a ranged style layout builder.
+    ///
+    /// Set `quantize` as `true` to have the layout coordinates aligned to pixel boundaries.
+    /// That is the easiest way to avoid blurry text and to receive ready-to-paint layout metrics.
+    ///
+    /// For advanced rendering use cases you can set `quantize` as `false` and receive
+    /// fractional coordinates. This ensures the most accurate results if you want to perform
+    /// some post-processing on the coordinates before painting. To avoid blurry text you will
+    /// still need to quantize the coordinates just before painting.
+    ///
+    /// Your should round at least the following:
+    /// * Glyph run baseline
+    /// * Inline box baseline
+    ///   - `box.y = (box.y + box.height).round() - box.height`
+    /// * Selection geometry's `y0` & `y1`
+    /// * Cursor geometry's `y0` & `y1`
+    ///
+    /// Keep in mind that for the simple `f32::round` to be effective,
+    /// you need to first ensure the coordinates are in physical pixel space.
```

Comment:
This is explanation is awesome. We should link to it from the CHANGELOG, and make it clear that the simplest upgrade path from `0.3` -> `0.4` is just to set `quantize` to true.

Reply:
I added a migration section to the changelog, linking to docs.rs which will eventually show this text.